### PR TITLE
Use multi-line print for all multi-line node types

### PR DESCRIPTION
**Pull request by mgrip**
Reusing solution from #14 for other functions that need to print multiple lines

### CODE REVIEW

**Comment by mgrip** [comment]
@czosel i just moved the functions you made to handle line spacing to the top w/ the other helper functions - but let me know if you think there's a better way to organize things!

File: src/printer.js [hunk 1]
```diff
@@ -1033,15 +1054,14 @@ function printNode(path, options, print) {
...
-                        lineShouldEndWithSemicolon(adaptationPath) ? ";" : ""
-                      ]);
-                    }, "adaptations")
-                  )
+                  concat([
+                    line,
+                    path.call(
+                      adaptationsPath =>
+                        printLines(adaptationsPath, options, print),
+                      "adaptations"
```
Comment (czosel) [inline src/printer.js]: indentation issue?

Comment (mgrip) [reply src/printer.js]: so this is actually how prettier formats it haha - `"adaptations"` is the second argument for `call()`, the inline arrow function is the 1st argument. I agree its a little muddy though...

Comment (czosel) [reply src/printer.js]: Oh, right ^_^

**Pull request merged**